include/sudoku.h

```diff
@@ -192,3 +216,36 @@ void col_groups(int column_group[][9], int b_1[9][3], int b_2[9][3], int b_3[9][
 void row_groups(int r_group[][9], int c_1[3][9], int c_2[3][9], int c_3[3][9]);
 
 #endif
+
+#ifndef INPUT_SUDOKU_H
+#define INPUT_SUDOKU_H
+/** 
+ * @brief This function removes surplus characters or symbols from the input buffer.
+ *        Otherwise, if restricted symbols or characters have been entered during the
+ *        call to fgets(), the surplus characters (after MAX_DIGITS chars) remain in the input buffer
+ *        and will be wrongly accepted as input on the next iteration of the loop.
+ *
+ */
+static inline void ClearInputBuffer();
+
+/** 
+ * @brief This function initially clears the input buffer and checks for restricted input values and returns only if the entered value 
+ *        is integer. 
+ *
+ * @param[in] *inputInteger --- This parameter takes the user input.
+ * 
+ */
+void getIntegerFromStdin(int *inputInteger);
```

Comment:
The functions should be named with lower case letters and spaces. The prof will deduct points for wrong standard.